app/scripts/modules/dcos/serverGroup/configure/wizard/templateSelection.controller.js

```diff
@@ -0,0 +1,74 @@
+'use strict';
```

Comment:
We consolidated all the provider-specific template selectors a few weeks ago to use the `deployInitializer` component in core - can you give that a shot and see if it works?